.github/CODEOWNERS

```diff
@@ -4,4 +4,4 @@
 # For syntax help see:
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 
-*     @jiren @skuruppu @dazuma @hengfengli @olavloite @xiangshen-dk
+*     @googleapis/yoshi-ruby @jiren @skuruppu @hengfengli @olavloite @xiangshen-dk
```

Comment:
I think that while we're editing this, we can also remove `jiren` from the list, as they are no longer involved in the development of this library.